Intersect.Server/Entities/Events/CommandProcessing.cs

```diff
@@ -568,7 +568,7 @@ Stack<CommandInstance> callStack
             }
             else
             {
-                player.EquipItem(ItemBase.Get(command.ItemId));
+                player.EquipItem(ItemBase.Get(command.ItemId), -1, command.TriggerCooldown);
```

Comment:
Instead of `, -1, command.TriggerCooldown` you should be able to do `, updateCooldown: command.TriggerCooldown`